paper_io_sim.tex

```diff
@@ -781,7 +796,9 @@
 
         \subsection{Single-threaded execution (Exp 1)}
 
-        The page cache simulation model drastically reduced I/O simulation
+        \todo[inline]{Henri: In the paragraph below I would remove the ``The first read was ... data." sentence.}
```

Comment:
not sure, I find it interesting to note that WRENCH's error is low when no caching is involved. I'll move the sentence later in the paragraph, I agree that it's not the first thing to look at.

Reply:
Right, my main complain was that the result discussion started with that.